template_config.yml

```diff
@@ -32,7 +33,7 @@ pulp_settings:
   ansible_api_hostname: http://pulp:80
   ansible_content_hostname: http://pulp:80/pulp/content
 pulpcore_branch: 3.8
-pulpcore_pip_version_specifier: "<3.10"
+pulpcore_pip_version_specifier: <3.10
```

Comment:
Does this work ok? I thought `<` was a special char in yaml.

Reply:
I don't know, plugin-template removed the quotes, I'll put them back to be sure